Slim/Plugin/AudioScrobbler/HTML/EN/plugins/AudioScrobbler/settings/basic.html

```diff
@@ -9,9 +9,24 @@
 		[% END %]
 
 		<div class="prefDesc">[% "SETUP_PLUGIN_AUDIOSCROBBLER_ACCOUNTS_USERNAME_PASSWORD" | string %]</div>
-		<input type="text" class="stdedit" name="pref_username" id="username" value="" size="20">
-		&nbsp;&nbsp;
-		<input type="password" class="stdedit" name="pref_password" id="password" value="" size="20">
+		<div style="margin-bottom:16px;">
+			<label for="username" style="margin-right:4px;">[% "SETUP_PLUGIN_AUDIOSCROBBLER_ACCOUNTS_USERNAME" | string %]</label>
+			<input type="text" class="stdedit" name="pref_username" id="username" value="" size="20">
+			&nbsp;&nbsp;
+			<label for="password" style="margin-right:4px;">[% "SETUP_PLUGIN_AUDIOSCROBBLER_ACCOUNTS_PASSWORD" | string %]</label>
+			<input type="password" class="stdedit" name="pref_password" id="password" value="" size="20">
```

Comment:
Try to wrap each label/input combination into a single `span` or something which then can be marked not to wrap inside. Right now these look a bit confusing on small screens (mobile) as the input field would be separated from its label.
<img width="472" height="216" alt="Bildschirmfoto 2025-08-19 um 06 34 53" src="https://github.com/user-attachments/assets/e7749005-5723-4bb8-ada1-3a71831647e8" />